### PR TITLE
Adds commenting and adjusts code format

### PR DESCRIPTION
**Pull request by neriyahbutler**
adds features that stores and displays comments and changes code to align with Google's code guides

### CODE REVIEW

File: portfolio/src/main/java/com/google/sps/servlets/DataServlet.java [hunk 1]
```diff
@@ -11,22 +11,54 @@
...
+    String json = convertToJson(commentArray);
+ 
+      response.setContentType("application/json");
+      response.getWriter().println(json);
+  }
+ 
+  @Override
+  public void doPost(HttpServletRequest request, HttpServletResponse response) throws IOException {
+    String comment = getComments(request);
+    commentArray.add(comment);
```
Comment (gwroblew) [inline portfolio/src/main/java/com/google/sps/servlets/DataServlet.java:42]: If invalid input is sent, an empty comment will be still added. The code should rather return the "BAD REQUEST" error.

File: portfolio/src/main/webapp/script.js [hunk 2]
```diff
@@ -11,18 +11,18 @@
...
-  greetingContainer.innerText = greeting;
+ 
+function getComments(){
+  fetch('/data').then(response => response.json()).then((text) => {
+    const commentContainer = document.getElementById('commentList');
+    const jsonValues = Object.values(text);
+ 
+    console.log(jsonValues);
+    
+    for(var i = 0; i < Object.values(text).length; i++){
```
Comment (gwroblew) [inline portfolio/src/main/webapp/script.js:22]: replace Object.values(text) with jsonValues?

**Review by gwroblew — APPROVED** [review]

File: portfolio/src/main/webapp/script.js (hunk 2 above)
Comment (neriyahbutler) [reply portfolio/src/main/webapp/script.js:22]: issue will be resolved in next pull request

File: portfolio/src/main/java/com/google/sps/servlets/DataServlet.java (hunk 1 above)
Comment (neriyahbutler) [reply portfolio/src/main/java/com/google/sps/servlets/DataServlet.java:42]: issue will be resolved in next pull request

**Pull request merged**